### ISSUE TITLE
Demo link not working

### ISSUE THREAD

**Issue by johannesschobel** [timcreative/paper-dashboard#26]
The Link to the Demo project is not working

**Comment by ppazos** [comment]
yes, it's broken https://demos.creative-tim.com/paper-dashboard-2/examples/dashboard.html

**Comment by groovemen** [comment]
Hello guys, 

Thank you for using our products, I will ask you to tell us where do you found this link. 
**for now, you can access the product at the following link: https://demos.creative-tim.com/paper-dashboard/examples/dashboard.html

Best, 
Stefan

**Comment by johannesschobel** [comment]
Dear Stefan, thanks a lot for the updated link!

The broken link is located within the `README.md` file. Its the headline `Paper Dashboard 2`!
Further, there is a `links` section in the readme file, that also contains the wrong link.. Hope this helps

**Comment by groovemen** [comment]
Thank you very much for letting us know about this issue. We will fix ASAP the problem. 

All the best, 
Stefan

**Comment by johannesschobel** [comment]
you're welcome!
